rct229/rules/section15.py

```diff
@@ -220,7 +220,7 @@ def get_calc_vals(self, context, data=None):
             return {
                 "baseline_transformer_efficiency": context.baseline["efficiency"],
                 "required_baseline_transformer_efficiency": table_8_4_4_eff(
-                    phase=baseline_phase, kVA=baseline_kVA
+                    phase=baseline_transformer_phase, kVA=baseline_transformer_kVA
```

Comment:
Good.  This is the same as the branch I'm currently working in. Not sure why it wasn't already this way.